Backend/BikesRentalServer.Services/MalfunctionsService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using BikesRentalServer.Infrastructure;
+using BikesRentalServer.Models;
+using BikesRentalServer.Services.Abstract;
+using BikesRentalServer.DataAccess.Repositories.Abstract;
+
+namespace BikesRentalServer.Services
+{
+    public class MalfunctionsService : IMalfunctionsService
+    {
+        private readonly UserContext _userContext;
+        private readonly IMalfunctionsRepository _malfunctionRepository;
+        private readonly IBikesRepository _bikesRepository;
+        private readonly IUsersRepository _usersRepository;
+        public MalfunctionsService(UserContext userContext, IMalfunctionsRepository malfunctionRepository, IBikesRepository bikesRepository, IUsersRepository usersRepository)
+        {
+            _malfunctionRepository = malfunctionRepository;
+            _userContext = userContext;
```

Comment:
Znowu moje upośledzenie się odzywa: userContext przekazujemy jako pierwszy parametr, to można go jako pierwszego przypisać... Ale to w ogóle bez znaczenia ze strony działania